moveit_core/CMakeLists.txt

```diff
@@ -44,8 +44,16 @@ endif()
 
 find_package(fcl QUIET)
 if (fcl_FOUND)
-  set(LIBFCL_LIBRARIES ${FCL_LIBRARIES})
-  set(LIBFCL_INCLUDE_DIRS ${FCL_INCLUDE_DIRS})
+  # convert cmake target to variables for catkin_package DEPENDS
+  if(TARGET ${FCL_LIBRARIES})
```

Comment:
This will break if `FCL_LIBRARIES` becomes a list of targets, right?

Reply:
why would that happen? Either it is a target or it is a list, or?

Reply:
If they decide in future versions of FCL to export multiple targets...

Reply:
I guess we can fix that in the future, or in other words let's fix this with moveit2 and ament ;).

Reply:
Ok, let's be optimistic that they don't change that soon ;-)